src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(double side, Color color) {
+        this.side = side;
+        super.setColor(color);
```

Comment:
I would recommend to create constructor in `Figure` class
`public Figure(color)`

and use it, when creating concrete figures. It will help us avoid having partial-stated objects and sounds more logical: when we create figure, it cannot be without color